### DISCUSSION TITLE
[Bug] Showcase graphs are all empty

### DISCUSSION THREAD

**Discussion by ceastman-r7** [apache/skywalking#8630] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

Showcase (apache/skywalking-showcase)

### What happened

I followed the directions here to install the showcase to a kubernetes cluster: https://skywalking.apache.org/docs/skywalking-showcase/latest/readme/#quick-start
I opened my browser to http://localhost:8080/ but all the graphs are empty, no matter which time range I select nor which feature I look at.

![Uploading Screen Shot 2022-03-07 at 10.37.16 AM.png…]()

### What you expected to happen

I would expect the showcase installation to produce data and populate the various dashboards and graphs like http://demo.skywalking.apache.org/

### How to reproduce

git clone https://github.com/apache/skywalking-showcase.git
cd skywalking-showcase
make deploy.kubernetes

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
I can't see your graph, but generally, you need to check 

1. Whether pods are up in k8s
2. What is the timezone setting on the UI, for the k8s, it should be UTC-0 and click the refresh button, to make sure it is on the latest time.

**Comment by wu-sheng** [comment]
> I would expect the showcase installation to produce data and populate the various dashboards and graphs like http://demo.skywalking.apache.org/

They are the same. We just host the showcase on our sponsor's GKE.

**Reply by ceastman-r7** [reply]
All the pods are up and running in k8s, ill check the time zone.

**Reply by ceastman-r7** [reply]
More details: AWS eks version 1.21. I am going to create a new cluster and only install the showcase to see if my original cluster is causing the problem.

**Reply by wu-sheng** [reply]
Try to check the data in your ElasticSearch, such as `service_traffic` index, which would provide more details I think. Also, OAP logs would be helpful

**Reply by ceastman-r7** [reply]
So I created a new cluster and did the make deploy.kubernetes (which was successful).

if i access the ui using this method the graphs are empty:
kubectl port-forward svc/rocket-bot 8080:8080 --namespace default

but i setup the ingress using this method: https://istio.io/latest/docs/tasks/traffic-management/ingress/kubernetes-ingress/#configuring-ingress-using-an-ingress-resource

and then i point my browser to the load balancer with its hostname the graphs are populated:

<img width="1791" alt="Screen Shot 2022-03-08 at 8 44 01 AM" src="https://user-images.githubusercontent.com/97614291/157261050-6e662869-9db1-4609-a583-55163af19d53.png">

vs

<img width="1792" alt="Screen Shot 2022-03-08 at 8 44 11 AM" src="https://user-images.githubusercontent.com/97614291/157261058-41109347-68d1-4491-82e5-a9272770a949.png">

**Reply by wu-sheng** [reply]
Your browser is on UTC-6(right top), but k8s is usually(default) running on UTC-0.
Change it on UI, I think you should be all good.

**Reply by ceastman-r7** [reply]
@wu-sheng the timezone was the issue, thank you. Maybe an update to the documentation?

**Reply by wu-sheng** [reply]
Feel free to send a pull request to that doc.
